src/components/run/InterpretationLog.tsx

```
@@ -175,67 +254,318 @@ export const InterpretationLog: React.FC<InterpretationLogProps> = ({ isOpen, se
+                                <TableCell 
+                                  key={column}
+                                  sx={{ 
+                                    borderBottom: 'none',
+                                    py: 2
+                                  }}
+                                >
+                                  {row[column]}
+                                </TableCell>
+                              ))}
+                            </TableRow>
+                          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)